dojo/jira_link/views.py

```diff
@@ -358,684 +404,644 @@ def delete_jira(request, tid):
-#                         "16x16":"http://www.testjira.com/secure/useravatar?size=xsmall&ownerId=valentijn&avatarId=11101",
-#                         "32x32":"http://www.testjira.com/secure/useravatar?size=medium&ownerId=valentijn&avatarId=11101"
-#                      },
-#                      "displayName":"Valentijn Scholten",
-#                      "active":true,
-#                      "timeZone":"Europe/Amsterdam"
-#                   },
-#                   "created":"2020-11-11T18:55:21.425+0100",
-#                   "updated":"2020-11-11T18:55:21.425+0100"
-#                }
-#             ],
-#             "maxResults":2,
-#             "total":2,
-#             "startAt":0
-#          },
-#          "worklog":{
-#             "startAt":0,
-#             "maxResults":20,
-#             "total":0,
-#             "worklogs":[
-#             ]
-#          }
-#       }
-#    },
-#    "comment":{
-#       "self":"http://www.testjira.com/rest/api/2/issue/89820/comment/456843",
-#       "id":"456843",
-#       "author":{
-#          "self":"http://www.testjira.com/rest/api/2/user?username=valentijn",
-#          "name":"valentijn",
-#          "key":"valentijn",
-#          "emailAddress":"valentijn.scholten@isaac.nl",
-#          "avatarUrls":{
-#             "48x48":"http://www.testjira.com/secure/useravatar?ownerId=valentijn&avatarId=11101",
-#             "24x24":"http://www.testjira.com/secure/useravatar?size=small&ownerId=valentijn&avatarId=11101",
-#             "16x16":"http://www.testjira.com/secure/useravatar?size=xsmall&ownerId=valentijn&avatarId=11101",
-#             "32x32":"http://www.testjira.com/secure/useravatar?size=medium&ownerId=valentijn&avatarId=11101"
-#          },
-#          "displayName":"Valentijn Scholten",
-#          "active":true,
-#          "timeZone":"Europe/Amsterdam"
-#       },
-#       "body":"test2",
-#       "updateAuthor":{
-#          "self":"http://www.testjira.com/rest/api/2/user?username=valentijn",
-#          "name":"valentijn",
-#          "key":"valentijn",
-#          "emailAddress":"valentijn.scholten@isaac.nl",
-#          "avatarUrls":{
-#             "48x48":"http://www.testjira.com/secure/useravatar?ownerId=valentijn&avatarId=11101",
-#             "24x24":"http://www.testjira.com/secure/useravatar?size=small&ownerId=valentijn&avatarId=11101",
-#             "16x16":"http://www.testjira.com/secure/useravatar?size=xsmall&ownerId=valentijn&avatarId=11101",
-#             "32x32":"http://www.testjira.com/secure/useravatar?size=medium&ownerId=valentijn&avatarId=11101"
-#          },
-#          "displayName":"Valentijn Scholten",
-#          "active":true,
-#          "timeZone":"Europe/Amsterdam"
-#       },
-#       "created":"2020-11-11T18:55:21.425+0100",
-#       "updated":"2020-11-11T18:55:21.425+0100"
-#    }
-# }
+# example incoming requests from JIRA Server 8.13.0
```

Comment:
Do we want maybe to move these example payloads away in specific files in the repos?

Reply:
That's an interesting point.  I was looking for some Jira examples and went to the tests.  I don't think it would have occurred to me to look in the views file TBH.

Reply:
I agree. Should be better to move these exemple as unit tests data.
Also a link to the current JIRA documentation could be usefull https://developer.atlassian.com/server/jira/platform/webhooks/

Reply:
if we create files we are not using it becomes confusing so that's why I didn't do that. but this PR doesn't change anything around examples, just the logic of the author name check. and some moving around of the example.

Reply:
but there are already examples in the unit tests code, so I will put a link to that then